SecureFirmwareDevelopmentBestPractices.md

```diff
@@ -149,7 +149,7 @@ At the time of writing, a complementary work is ongoing in the Open Compute Proj
 [OpenCompute Project Contributions, IBM White Paper, "Best Practices for Firmware Code Signing"](http://files.opencompute.org/oc/public.php?service=files&t=f4171bae8c7a32f05b0401378ee08483&download)[^3]
 
 
-    "Out of scope for this paper are chain of trust requirements, firmware development practices, supply chain and manufacturing processes, as well as usage of certificates, certificate authorities, and trusted application stores. Also out of scope for this paper but needing attention, is establishing trust in audit logs. Are the audit logs protected against tampering? Can the audit logs be falsified from the start? Are automated procedures in place to detect anomalous activity related to signing server audit logs?"
+&emsp;&emsp;"Out of scope for this paper are chain of trust requirements, firmware development practices, supply chain and manufacturing processes, as well as usage of certificates, certificate authorities, and trusted application stores. Also out of scope for this paper but needing attention, is establishing trust in audit logs. Are the audit logs protected against tampering? Can the audit logs be falsified from the start? Are automated procedures in place to detect anomalous activity related to signing server audit logs?"
```

Comment:
Lets avoid manual spacing and adopt the typical GitHub markdown layout options.

Reply:
Agree this is not ideal, but I didn't find a "typical GitHub layout" way to do this. Feel free to suggest the right way.

Reply:
Found a better way, and also fixed some another list where bold text wasn't properly marked.